components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/org/wso2/carbon/policyeditor/ui/i18n/JSResources.properties

```diff
@@ -1,3 +1,21 @@
+#
+# Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```

Comment:
This file has been created and added in 2014, should I change year? if so what should I use?